optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -37,36 +44,35 @@ export const GraphHistory: FC<{
   const [filterCompleteTrial, setFilterCompleteTrial] = useState<boolean>(false)
   const [filterPrunedTrial, setFilterPrunedTrial] = useState<boolean>(false)
 
-  const [targets, selected, setTarget] = useObjectiveAndUserAttrTargets(study)
-  const trials = useFilteredTrials(
-    study,
+  const [targets, selected, setTarget] = useObjectiveAndUserAttrTargets(
+    studies ? studies[0] : null
```

Comment:
[Just FYI] After applied my following suggestions, you need to update this line.

https://github.com/optuna/optuna-dashboard/pull/380/files#r1118855931